mmv1/products/memorystore/Instance.yaml

```diff
@@ -155,6 +155,57 @@ properties:
               "Output only. Target number of replica nodes per shard for the
               instance. "
             output: true
+  - name: 'automatedBackupConfig'
+    type: NestedObject
+    description: "The automated backup config for a instance."
+    custom_flatten: 'templates/terraform/custom_flatten/memorystore_instance_automated_backup_config.go.tmpl'
+    custom_expand: 'templates/terraform/custom_expand/memorystore_instance_automated_backup_config.go.tmpl'
+    properties:
+      # - name: 'automatedBackupMode'
```

Comment:
this should be removed, is this an existing field in the API that provided to be difficult?